app/api/schema/feedbacks.py

```diff
@@ -23,7 +23,7 @@ class Meta:
         inflect = dasherize
 
     id = fields.Str(dump_only=True)
-    rating = fields.Float(required=True, validate=Range(min=0, max=5))
+    rating = fields.Float(required=True, validate=Range(min=1, max=5))
```

Comment:
This behaviour of frontend deleting the record when setting the rating to 0 is wrong IMO. Delete action should be very explicit. Either show a dialog on frontend that setting rating to 0 will delete the feedback

Reply:
FE is using ui-rating from semantic UI. If a user selects rating 2 suppose and wants to clear out the rating, he/she needs to click on 2 stars again and the rating is cleared (set to 0). I think showing a dialog box will be better.
@CosmicCoder96 your opinion?

Reply:
@iamareebjamal please review this PR. The FE PR is still WIP so I can include your suggestions there